site/src/Tour.jsx

```diff
@@ -0,0 +1,175 @@
+    placement: "auto",
+    offset: 0,
+  },
+  {
+    target: ".maplibregl-ctrl-top-right",
+    content: "These tools faciliate navigation of the world map.",
+    disableBeacon: true,
+    title: "Map Navigation Tools",
+    placement: "auto",
+    offset: 0,
+  },
+  {
+    target: ".inspector-panel",
+    content:
+      "This is the inspector panel. Clicking on features in the world will display them here. These features are only clickable after a zoom threshold has been reached.",
+    disableBeacon: true,
+    title: "Inspector Panel",
+    placement: "right",
+    offset: 0,
+  },
+  {
+    target: ".maplibregl-ctrl-bottom-right",
+    content:
+      "This is the map footer. Credits and copyrights can be found here.",
+    disableBeacon: true,
+    title: "Map Footer",
+    placement: "top-end",
+    offset: -10,
+  },
+];
+const sampleFeature = {
+  geometry: {
+    type: "Point",
+    coordinates: [3.7302714586257935, 51.05027395815554],
+  },
+  type: "Feature",
+  properties: {
+    id: "08f194db132d2b6d0388899915aac1fc",
+    "@name": "Grill Mix Centrum",
+    "@category": "bar_and_grill_restaurant",
+    names: '{"primary":"Grill Mix Centrum","common":null,"rules":null}',
+    confidence: 0.9584614231086451,
+    categories:
+      '{"main":"bar_and_grill_restaurant","alternate":["pizza_restaurant","doner_kebab"]}',
+    websites: '["http://www.pizzacity.be"]',
+    socials: '["https://www.facebook.com/612060042296776"]',
+    phones: '["+3292257440"]',
+    addresses:
+      '[{"freeform":"Vlaanderenstraat 85","locality":"Gent","postcode":"9000","region":null,"country":"BE"}]',
+    version: 0,
+    update_time: "2024-04-11T00:00:00.000Z",
+    sources:
+      '[{"property":"","dataset":"meta","record_id":"612060042296776","confidence":null}]',
+  },
+  id: 38848842,
+  layer: {
+    id: "places",
+    type: "circle",
+    source: "overture-places",
+    "source-layer": "places",
+    filter: [">=", ["get", "confidence"], 0],
+    layout: {},
+    paint: {
+      "circle-color": {
+        r: 0.792156862745098,
+        g: 0.6980392156862745,
+        b: 0.8392156862745098,
+        a: 1,
+      },
+      "circle-radius": 1.8749883174673414,
+      "circle-stroke-width": 2,
+      "circle-stroke-color": {
+        r: 0,
+        g: 0,
+        b: 0,
+        a: 1,
+      },
+    },
+  },
+  source: "overture-places",
+  sourceLayer: "places",
+  state: {},
+};
+
+function Tour({ run, modeName, setMapEntity }) {
+  const [stepIndex, setStepIndex] = useState(0);
+
+  const stepBGColor =
+    modeName === "theme-dark" ? "dimgray" : "var(--ifm-color-secondary-light)";
+  const stepTextColor =
+    modeName === "theme-dark" ? "var(--ifm-color-secondary-light)" : "black";
+
+  const handleJoyrideCallback = (event) => {
+    console.log(event);
+    if ([EVENTS.STEP_AFTER, EVENTS.TARGET_NOT_FOUND].includes(event.type)) {
+      const nextStepIndex =
+        event.index + (event.action === ACTIONS.PREV ? -1 : 1);
+      if (
```

Comment:
Switching from step 4 to 5 means we need to open the inspector panel. This also requires a slight delay of the tour so that the target exists when the tour attempts to advance.